source/wp-content/themes/wporg-developer-2023/patterns/single-github-handbook.php

```diff
@@ -0,0 +1,94 @@
+<?php
+/**
+ * Title: Single Handbook Template (GitHub)
+ * Slug: wporg-developer-2023/single-github-handbook
+ * Inserter: no
+ */
+
+?>
+
+<!-- wp:group {"tagName":"main","layout":{"type":"constrained","wideSize":"1280px","contentSize":"680px"},"style":{"spacing":{"padding":{"left":"var:preset|spacing|edge-space","right":"var:preset|spacing|edge-space"}}}} -->
+<main class="wp-block-group alignfull" style="padding-left:var(--wp--preset--spacing--edge-space);padding-right:var(--wp--preset--spacing--edge-space)">
+
+	<!-- wp:group {"className":"has-three-columns","layout":{"type":"constrained","justifyContent":"left"},"align":"wide","style":{"spacing":{"margin":{"top":"var:preset|spacing|edge-space"}}}} -->
+	<div class="wp-block-group alignwide has-three-columns" style="margin-top:var(--wp--preset--spacing--edge-space)">
+
+		<!-- wp:wporg/chapter-list {"style":{"spacing":{"margin":{"bottom":"80px"}}}} /-->
+
+		<!-- wp:group {"tagName":"article","style":{"spacing":{"margin":{"top":"0px"}}}} -->
+		<article class="wp-block-group" style="margin-top:0px">
+			<!-- wp:post-title {"level":1,"style":{"spacing":{"margin":{"bottom":"var:preset|spacing|40"}}}} /-->
+
+			<!-- wp:group {"tagName":"aside","className":"sidebar-container"} -->
+			<aside class="wp-block-group sidebar-container">
+				<!-- wp:search {"label":"<?php esc_attr_e( 'Search', 'wporg' ); ?>","showLabel":false,"widthUnit":"px","buttonText":"<?php esc_attr_e( 'Search', 'wporg' ); ?>","buttonPosition":"button-inside","buttonUseIcon":true,"className":"is-style-secondary-search-control","borderColor":"light-grey-1","placeholder":"<?php esc_attr_e( 'Search in the documentation', 'wporg' ); ?>"} /-->
+
+				<!-- wp:wporg/table-of-contents {"style":{"spacing":{"padding":{"top":"var:preset|spacing|20","right":"var:preset|spacing|20","bottom":"var:preset|spacing|20","left":"var:preset|spacing|20"}}}} /-->
+
+				<!-- wp:paragraph {"fontSize":"small","className":"is-link-to-top"} -->
+				<p class="has-small-font-size is-link-to-top"><a href="#wp--skip-link--target"><?php esc_html_e( '↑ Back to top', 'wporg' ); ?></a></p>
+				<!-- /wp:paragraph -->
+			</aside>
+			<!-- /wp:group -->
+
+			<!-- wp:post-content {"layout":{"inherit":true}} /-->
+
+			<!-- wp:group {"style":{"spacing":{"padding":{"top":"var:preset|spacing|40"},"margin":{"top":"var:preset|spacing|40","bottom":"var:preset|spacing|40"},"blockGap":"var:preset|spacing|20"},"border":{"top":{"color":"var:preset|color|light-grey-1","width":"1px"}}},"layout":{"type":"flex","flexWrap":"nowrap","justifyContent":"left"}} -->
```

Comment:
This markup will also be needed in `single.html`. What's the best way to share it?

[Figma](https://www.figma.com/file/2WxlJFzMJvqPfZL1EkAOVp/Developer-Resources?node-id=1160%3A10750&t=8Xx4DYGn3OnIuhLv-0)

<img width="477" alt="Screen Shot 2023-02-02 at 9 04 34 AM" src="https://user-images.githubusercontent.com/1657336/216196730-2cc4a99a-51bd-4981-a1ab-b8da7efad150.png">

Reply:
Tried a different approach to the patterns & reusable content in #196, and that would let us share the meta pattern between `single.html` & `single-handbook.html`.